lib/iframe-browser.js

```diff
@@ -128,6 +138,37 @@ define(function (require, exports, module) {
         }
     }
 
+    /**
+     * Used to show second pane, change lilveDevButton background and close the detached preview
+     */
+    function attachPreview() {
+        if(detachedWindow && isReload) {
+            isReload = false;
+            return;
+        }
+
+        if(getDetachedPreview()) {
+            detachedWindow.removeEventListener("beforeunload", attachPreview, false);
+            detachedWindow.close();
+            detachedWindow = null;
```

Comment:
Why do you do this?  I think that's wrong.  `getDetachedPreview()` will return nothing if it's `.closed===true`.